bootstrap3-components/src/main/resources/bootstrap3nt_modal/html/modal.jsp

```diff
@@ -22,9 +22,15 @@
     <c:set var="size" value="modal-${currentNode.properties.size.string}"/>
 </c:if>
 <c:set var="title" value="${currentNode.properties['jcr:title'].string}"/>
+<c:set var="openOnPageView" value="${currentNode.properties['openOnPageView'].boolean}"/>
+<c:set var="hideOpenButton" value="${currentNode.properties['hideOpenButton'].boolean}"/>
 
 <!-- Button trigger modal -->
-<button type="button" class="btn btn-${state} btn-lg" data-toggle="modal" data-target="#modal_${currentNode.identifier}">
+<button type="button"
```

Comment:
I think you should wrap the button around the <c:if test="${!renderContext.editMode and hideOpenButton}"> instead of adding style to the button, as anyway it won't show until you modify and then republish the node

Reply:
ME: Brain COME HEEEEERE 😤!!!!
Brain: No. 

I agree, thank you for the review